server/main.lua

```diff
@@ -57,38 +59,32 @@ RegisterNetEvent('qb-tow:server:11101110', function(drops)
 
     local playerPed = GetPlayerPed(src)
     local playerCoords = GetEntityCoords(playerPed)
-    if Player.PlayerData.job.name ~= "tow" or #(playerCoords - vector3(Config.Locations["main"].coords.x, Config.Locations["main"].coords.y, Config.Locations["main"].coords.z)) > 6.0 then
+    if Player.PlayerData.job.name ~= "tow" or #(playerCoords - vec3(Config.Locations["main"].coords.x, Config.Locations["main"].coords.y, Config.Locations["main"].coords.z)) > 6.0 then
         return DropPlayer(src, Lang:t("info.skick"))
     end
 
     drops = tonumber(drops)
     local bonus = 0
     local DropPrice = math.random(150, 170)
     if drops > 5 then
-        bonus = math.ceil((DropPrice / 10) * 5)
-    elseif drops > 10 then
-        bonus = math.ceil((DropPrice / 10) * 7)
-    elseif drops > 15 then
-        bonus = math.ceil((DropPrice / 10) * 10)
-    elseif drops > 20 then
-        bonus = math.ceil((DropPrice / 10) * 12)
+        if drops > 20 then drops = 20 end
+        bonus = math.ceil((DropPrice / 10) * ((3 * (drops / 5)) + 2))
```

Comment:
this piece of logic is a bit confusing. Why is drops maxing at 20 only if drops is > 5?

Reply:
It maxes out at 20 to (kind of) mimic the original system.
![Screenshot_20221225_131211](https://user-images.githubusercontent.com/13180679/209469269-2647ac5c-c8e3-4a99-bb5c-4076de7af842.png)
linear scale.
Consider the line graph above, without the 20 limit, the graph would considerably grow, hence mimicing the original system by outputting the same value for everything over 20.